### DISCUSSION TITLE
Incorrect python version check

### DISCUSSION THREAD

**Discussion by zobelb** [aeon-toolkit/aeon#1371] [Q&A]
I'm trying to use this library inside of a tensorflow container and I'm getting an error on library version (I'm guessing this is a bug, but I haven't checked the bug list yet to see if it is already listed).

```
aeon version is 0.7.1
An exception occurred: IndividualInceptionClassifier requires python version to be <3.12, but system python version is 3.11.0rc1 (main, Aug 12 2022, 10:02:14) [GCC 11.2.0].
```

Does anyone know how I get around this? Changing python inside of the tensorflow container isn't straightforward and I don't have an option to deliver a non-containerized version.

**Comment by MatthewMiddlehurst** [comment]
Apologies for the late reply! Discussion for this is in issue #1395. I need to set up a better way for tracking discussions 🙂.

**Comment by zobelb** [comment]
I added that particular bug report after asking the question here.

**Reply by MatthewMiddlehurst** [reply]
Yeah I saw, thanks for that. We have a bot to post new issues, but that does not work for discussions apparently 🙁.
